repair/repair.cc

```diff
@@ -343,32 +343,35 @@ void tracker::done(repair_uniq_id id, bool succeeded) {
     }
     _done_cond.broadcast();
 }
-repair_status tracker::get(int id) const {
-    if (id >= _next_repair_command) {
+
+repair_status tasks::repair_module::get(int id) const {
+    if (id > _sequence_number) {
```

Comment:
why `>` and not `>=`?

Reply:
`_next_repair_command` is a number of next repair command while `_sequence_number` keeps the value of the last number assigned to some task.
I haven't changed that in `tasks::repair_module::repair_await_completion`, though. Will do.